src/Functions/FunctionsHashing.h

```diff
@@ -1193,6 +1196,11 @@ class FunctionAnyHash : public IFunction
     {
         for (size_t i = 0, size = column->size(); i < size; ++i)
         {
+            if (column->isNullAt(i))
+            {
+                vec_to[i] = static_cast<ToType>(0xe28dbde7fe22e41c);
```

Comment:
let's call `apply` on an empty `bytes` instead of hardcoding a constant

Reply:
The result of applying on empty bytes is not equal to result of hash(). And I should make the 0xe28dbde7fe22e41c as a macro value.

Reply:
@nickitat What do you think?